sklearn/tests/test_common.py

```diff
@@ -87,13 +88,20 @@ def _tested_estimators():
 
 
 @parametrize_with_checks(_tested_estimators())
-def test_estimators(estimator, check):
+def test_estimators(estimator, check, request):
```

Comment:
request is a [built-in pytest fixture](https://docs.pytest.org/en/latest/reference.html#request) providing information of the requesting test function.